src/common/copilot/PowerPagesCopilot.ts

```diff
@@ -68,16 +68,28 @@ export class PowerPagesCopilot implements vscode.WebviewViewProvider {
 
 
     if (SELECTED_CODE_INFO_ENABLED) { //TODO: Remove this check once the feature is ready
+
+        const handleSelectionChange = async (commandType: string) => {
+            const editor = vscode.window.activeTextEditor;
+            if (!editor) {
+                return;
+            }
+            const selectedCode = getSelectedCode(editor);
+            const selectedCodeLineRange = getSelectedCodeLineRange(editor);
+            if(commandType === "explainCode" && selectedCode.length === 0) {
```

Comment:
can we make the text case-insensitive like both upper or lowercase and even trim spaces

Reply:
Added to constant, both the places same constant will be used. So, now we don't require trim and case handling.